yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -89,10 +92,14 @@ export class AztecNodeService implements AztecNode {
     const p2pClient = await createP2PClient(config, new InMemoryTxPool(), archiver);
 
     // now create the merkle trees and the world state syncher
-    const merkleTreesDb = levelup(createMemDown());
-    const merkleTrees = await MerkleTrees.new(merkleTreesDb, await CircuitsWasm.get());
+    const db = levelup(
```

Comment:
I don't think I would do it like this. Every time the contract changes we leave behind a load of stale data. Can we have the world state create a sub-directory under the DATA_DIRECTORY and in the new() function wipe the directory contents if the rollup contract is different to that stored?